components/content/index.css

```
@@ -1,21 +1,42 @@
 .content {
     background-color: var(--blueviolet);
+    border-radius: 12px;
 }
 
 .content-holder {
     display: flex;
     height: 100%;
-    background-image: linear-gradient(var(--red), var(--green));
+    border-radius: 12px;
 }
 
 /* Content aside */
 
 .content-aside {
-    width: 20%;
+    width: 40%;
```

```suggestion
    width: 443px;
    min-width: 443px;
```

For now, we use dimensions strictly from the design